express-post-json/index.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const app = express();
+let nextId = 1;
+
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const gradesArray = [];
+  for (const key in grades) {
+    gradesArray.push(grades[key]);
+  }
+  res.json(gradesArray);
+});
+
+express.json();
+app.use(express.json());
+
+app.post('/api/grades', (req, res) => {
+  req.body.id = nextId;
+  grades[nextId] = req.body;
+  nextId++;
+  res.status(201).send(grades);
```

Comment:
The spec says "The server should respond with the same grade object it received, plus its `id` using the status code `201`." which means just the newly-created grade should be returned, not the entire set of grades.

Reply:
![image](https://user-images.githubusercontent.com/117887912/215218314-a5450da3-f5f6-4040-9a63-de06d6f13b35.png)